setup.py

```diff
@@ -85,6 +85,13 @@ def get_cffi_modules():
             cffi_modules.append(
                 'libqtile/widget/pulseaudio_ffi.py:pulseaudio_ffi'
             )
+    try:
+        import wlroots.ffi_build
+        cffi_modules.append(
+            'libqtile/backend/wayland/libinput_ffi_build.py:libinput_ffi',
+        )
+    except ImportError:
+        pass
```

Comment:
Do you want to include a message here (similar to pulse) that libinput support is not available (similar to the pulseaudio message)?

Reply:
Good idea - added